Ports/python-3.6/package.sh

```diff
@@ -1,11 +1,26 @@
 #!/bin/bash ../.port_include.sh
+
+source version.sh
+
 port=python-3.6
 version=3.6
 workdir=Python-3.6.0
 useconfigure=true
 configopts="--build=i686 --without-threads --enable-optimizations"
 makeopts="-j$(nproc) build_all"
 installopts="-j$(nproc) build_all"
-files="https://www.python.org/ftp/python/3.6.0/Python-3.6.0.tar.xz Python-3.6.0.tar.xz"
+files="${PYTHON_URL} ${PYTHON_ARCHIVE}"
 
 export CONFIG_SITE=$(pwd)/config.site
+
+if [ -x "$(command -v python3)" ]; then
```

Comment:
This checks for `python3` in user's `PATH` (i.e. only system's version of Python), not in the `PREFIX` that `BuildPython.sh` builds it to.

Reply:
The package scripts include the parent folders .port_include.sh script (see shebang line) and this script assures SERENITY_ROOT is set. Basically it must assure that Toolchains UseIt.sh is sourced, otherwise the compiler for cross compilation wouldn't be found.
When I looked at PATH after UseIt.sh, it shows: 
`/home/ema/checkout/serenity/Toolchain/Local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin`
For me this looks good so far. The $(command -v python3) also returns the path to the python3 built by serenity toolchain: If I run it in shell (with UseIt.sh sourced), it prints out: `/home/ema/checkout/serenity/Toolchain/Local/bin/python3`

Reply:
Ah, alright then :)